config.default.php

```diff
@@ -237,3 +237,10 @@
 
 // TOTP (2FA) issuer name
 define('TOTP_ISSUER', 'Kanboard');
+
+// default filter expression
+define('DEFAULT_FILTER', 'status:open');
+
+// an expression that will be added to the predefined filters
+// define('ADDITIONAL_FILTER', 'board_date:<=today');  // use this when plugin TaskBoardDate is installed
+define('ADDITIONAL_FILTER', '');  // use this otherwise
```

Comment:
Kanboard should not be coupled and depends of any plugin.

Another way to do it is to offer the possibility to configure those filters via the user interface instead of using PHP constants. In this way, people do not have to manually change a config file on the server.

In the templates, you could use the helper `$app->config('my_config_param', 'some default value')`. Add new form fields in this template: https://github.com/kanboard/kanboard/blob/master/app/Template/config/project.php